operator/v1/types_ingress.go

```diff
@@ -233,8 +233,36 @@ type LoadBalancerStrategy struct {
 	// +kubebuilder:validation:Required
 	// +required
 	Scope LoadBalancerScope `json:"scope"`
+	// aws provides configuration settings that are specific to AWS
+	// load balancers.
+	//
+	// If this field is empty, the default values are used.
+	//
+	// +optional
+	AWS *AWSLoadBalancerParameters `json:"type,omitempty"`
```

Comment:
Need to update the json tag: `type` → `aws`.

Reply:
@Miciah thanks for catching this. Commit `833cbd4` fixes the `AWS` json tag.